Assets/DirectionButtons.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using System;
+using UnityEngine.InputSystem;
+
+//attach script to player object
+
+public class DirectionButtons : MonoBehaviour
+{
+
+    //variables
+    private float movementSpeed = 5f;
+    private float rotationSpeed = 80f;
+    public Button forwardButton;
```

Comment:
We should make these variables private and use [Serializable] attribute so we can assign it in unity. If we leave them as public other the it leaves them open to manipulation from other aspects of the game